realm/realm-library/src/androidTestObjectServer/java/io/realm/CredentialsTests.java

```diff
@@ -27,6 +27,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
+
```

Comment:
Fixed.  I would like to point out that his stuff is making it past both checkstyle and lint!!